docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,37 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
+      -d, --driver=[type]      Driver to manage the network
```

Comment:
Reading the `remote` option, I may be mistaken here; although it's not clear to me how to specify *which* remote driver I want to use; can I have multiple plugins? If so, can I specify which plugin to use?